cmd/ddev/cmd/composer-create.go

```diff
@@ -154,6 +154,9 @@ ddev composer create --prefer-dist --no-interaction --no-dev psr/log
 // when they try ddev composer create-project
 var ComposerCreateProjectCmd = &cobra.Command{
 	Use: "create-project",
+	FParseErrWhitelist: cobra.FParseErrWhitelist{
```

Comment:
I just want to know how you figured out how to do this!

Reply:
Well, it's used in other commands too and I've had also a look at the cobra library to verify it.